### PR TITLE
Update plans to new TESTed testplans

### PR DESCRIPTION
**Pull request by bsels**
Only merge PR after TESTed master branch is updated to https://github.com/dodona-edu/universal-judge/tree/ft/seperation-master

### CODE REVIEW

**Review by charvp — APPROVED** [review]
LGTM, but let's wait until tonight or tomorrow to merge everything (semi-)simultaneously, since there are people using these exercises right now.

**Review by niknetniko — APPROVED** [review]

**Pull request merged**
